docs/v1.md

```diff
@@ -288,11 +308,45 @@
 - filterBy(filter: `(operation: Operation) => boolean`): `Operation[]`
 - filterBySend(): `Operation[]`
 - filterByReceive(): `Operation[]`
+- filterByTags(tags: `string[]`): `Operation[]`
 - get(id: `string`): `Operation` | `undefined`
 - has(id: `string`): `boolean`
 
+## OperationReply
+- id(): `string` | `undefined`
+- hasAddress(): `boolean`
+- address(): `OperationReplyAddress` | `undefined`
+- hasChannel(): `boolean`
+- channel(): `Channel` | `undefined`
+- extensions(): `Extensions`
```

Comment:
missing `messages()` and `hasMessages()`. See https://github.com/asyncapi/spec/pull/847/files#diff-34bca0c91901dffce093114c7d4cac16477410fbca7cabf3a1b888d7dfc3eb4dR980

Reply:
Yep, still missing 😄

Reply:
Added `messages(): Messages`. `hasMessages()` is not needed since `Messages` already have all necessary for that purpose.